faqs.rst

```diff
@@ -84,6 +84,29 @@ The upper limit to the number of open files is low.
 Increase the upper limit by adding a command ``ulimit -S -n 2048`` to you ~/.profile.
 Don't forget to re-login for the changes to take effect.
 
+Why I add custom gradle task like document, but gradle can't execute?
```

Comment:
"Why can't gradle execute gauge specs, despite adding a task as per the document?"